### DISCUSSION TITLE
Be able to limit size of custom question fields

### DISCUSSION THREAD

**Discussion by doegox** [pretix/pretix#4251] [Ideas]
Hello, 
For your open source conference, we're using Pretix data to automate the generation of badges and we let the opportunity to people to specify a custom field, such as nickname, X handle, affiliation...
We ask in the description to not write more than N chars but of course 10% don't read instructions...
It would be really nice to be able to configure and enforce a maximum size of the custom question fields

**Comment by raphaelm** [comment]
We don't think this is an adequate solution for the problem.

1) Badge layout at most conferences is decided much later than data entry

2) Number of characters is a bad proxy for horizontal width (ww is wider than lili in many fonts)

Most pretix users use pretix to generate the badges (may I humbly ask why you don't?) and I'm just right now working on a PR that allows automatically adjusting font size so every name fits the available width

**Comment by doegox** [comment]
Hello Raphaël
I'm doing lasered wood badges, cf e.g. last year https://www.thingiverse.com/thing:6113649
Adjusting font size is a good start but when you tell people to type max 14 chars and they type 40 :man_facepalming: that requires manual drastic adjustments anyway
